test/downloadurl.test.ts

```diff
@@ -12,7 +12,7 @@ describe('downloadUrl', () => {
         await downloadUrl.download();
 
         expect(fileExistsSync(downloadUrl.filepath)).to.be.true;
-    });
+    }, 10000);
```

Comment:
Why so much delay?

Reply:
Don't know. I saw that sometimes it fails.